### ISSUE TITLE
Middle-Mouse Button / Rotation not working under Parallels VM (M1 macOS)

### ISSUE THREAD

**Issue by icetear** [MaFi-Games/Captain-of-Industry-issues#179]
**Describe the issue**
When playing under Windows 11 ARM on M1 Parallels Desktop VM, the middle mouse button or any other means of rotating the camera is not working. It might be a problem with how the axis are defined in CoI - in Anno 1800 the middle mouse button works e.g.

**To Reproduce**
1. Start a new game
2. Press middle mouse button and rotate / use Option + Left Mouse Button etc. -> the cursor changes to rotation mode
3. Camera does not rotate

**Expected behavior**
The camera should rotate when recognizing movement while pushing MMB or Option.

**Additional context**
 - Game version: 0.4.1

**Attachments**

**Comment by ouTPhaze** [comment]
This issue also exists in Windows when using Parsec Gaming to play the game.

**Comment by HenryLoenwind** [comment]
And when using Moonlight (NVIDIA) and even VNC.

The cursor shape goes "rotate" when holding alt or the middle mouse button, but moving the mouse just moves the cursor.

**Comment by ouTPhaze** [comment]
Yup! Same here! Whether I hold middle mouse or alt, it shows the "rotate mode" cursor, but will not rotate.

**Comment by MaFi-Games** [comment]
Did this improve in any way after added key-bindings? Are you able to remap the middle mouse button?

**Comment by HenryLoenwind** [comment]
> Did this improve in any way after added key-bindings? Are you able to remap the middle mouse button?

It isn't that the button isn't detected, it's that moving the mouse while it is pressed does nothing---it just moves the cursor around. So the issue is the code that detects mouse movements and translates it into map rotation while preventing the cursor from moving. That's doing neither.
